### ISSUE TITLE
Error! Cannot read property 'total' of undefined

### ISSUE THREAD

**Issue by andylolz** [datopian/data-cli#350]
For `data-cli` versions 0.9.3 to 0.9.5, when I run:

```
data push --public
```

I see:

```
> Error! Cannot read property 'total' of undefined
```

See for example:
https://travis-ci.org/datasets/dac-and-crs-code-lists/builds/603313715#L214-L216

**Comment by anuveyatsu** [comment]
Hi @andylolz thanks for reporting this. It should be fixed in v0.9.6 which is available via NPM now.

**Comment by andylolz** [comment]
Works for me! Thanks
